### PR TITLE
Remove advice to use dynamic versions for dependencies

### PR DESCRIPTION
**Pull request by peter-tackage**

### CODE REVIEW

**Comment by jonikarppinen** [comment]
Btw, I think it's redundant to merge the branch to your local master first. Just create a branch on your fork, and then directly from that branch create PR against futurice:master (the default option). Just for future changes :)

File: README.md [hunk 1]
```diff
@@ -134,19 +134,17 @@ signingConfigs {
 }
 ```
 
-**Prefer Maven dependency resolution instead of importing jar files.** If you explicitly include jar files in your project, they will be of some specific frozen version, such as `2.1.1`. Downloading jars and handling updates is cumbersome, this is a problem that Maven solves properly, and is also encouraged in Android Gradle builds. You can specify a range of versions, such as `2.1.+` and Maven will handle the automatic update to the most recent version matching that pattern. Example:
+**Prefer Maven dependency resolution instead of importing jar files.** If you explicitly include jar files in your project, they will be of some specific frozen version, such as `2.1.1`. Downloading jars and handling updates is cumbersome, this is a problem that Maven solves properly, and is also encouraged in Android Gradle builds. For example:
```
Comment (jonikarppinen) [inline README.md]: Can you restore `groovy` here? It added syntax highlighting for this block.

File: README.md [hunk 2]
```diff
@@ -134,19 +134,17 @@ signingConfigs {
...
 
-```groovy
+```
```
Comment (jonikarppinen) [inline README.md]: Can you restore groovy here? It added syntax highlighting for this block.

**Comment by jonikarppinen** [comment]
Also, totally minor nitpick :) Your commit comment would have been better without the "Update README.md" in the beginning. That is kind of obvious when it's the only file you change + it hides the real beef of the commit message (in many views).

**Comment by peter-tackage** [comment]
No worries. Minor nitpicks are important! This case, I foolish accepted the default commit comment that Github provides and I'll reduce the branching/merging noise next time.

**Comment by jonikarppinen** [comment]
Looking good now!

**Comment by staltz** [comment]
:+1: to this PR and discussion!

**Pull request merged**